[CUBRIDMAN-259] Modify explanation of auto_restart_server parameter (#572)
CUBRID/cubrid-manual: en/admin/config.rst

**auto_restart_server**

>     **auto_restart_server** is a parameter to configure whether to restart the process when it terminates abnormally due to fatal errors being occurred in database server process (cub_server). This parameter is only available on Linux environment.

Reviewer: ```suggestion
    **auto_restart_server** is a parameter to configure whether to restart the process when it terminates abnormally due to fatal errors being occurred in database server process (cub_server). The default value is **yes**.
```

Author: 수정하였습니다.

>     **auto_restart_server** is a parameter to configure whether to restart the process when it terminates abnormally due to fatal errors being occurred in database server process (cub_server). This parameter is only available on Linux environment.

Reviewer 2: ```suggestion
    **auto_restart_server** is a parameter used to configure whether to restart the process when it terminates abnormally due to fatal errors occurring in the database server process (cub_server). This parameter is only available in Linux environments.

```

Author: 수정하였습니다.

**auto_restart_server**

>     **auto_restart_server** is a parameter to configure whether to restart the process when it stops due to fatal errors being occurred in database server process. This parameter is only available on Linux/UNIX environment.

Reviewer 3: 한글 매뉴얼 처럼 UNIX 제거

Author: 수정하였습니다.

>     If **auto_restart_server** value is set to **yes**, the master process automatically restarts the non-HA server process when it has stopped due to errors. It does not restart if the server stops through a normal process (by using the **STOP** command) or if it has already been restarted by this functionality within the past 120 seconds.

Reviewer 3: 한글 수정한 부분을 영문에도 반영해주세요.

Author: 현재까지 나온 모든 수정 사항들 영문 매뉴얼에 반영하였습니다.
